### PR TITLE
7Task. Andrei Lebedev

### PR DESCRIPTION
**Pull request by lebedos2103**
https://travis-ci.org/github/lebedos2103/js-assignments/builds/720949973

### CODE REVIEW

**Comment by vramaniuk** [comment]
The file 01-strings-tasks.js shouldn't be changed in this PR.
Please, try to perform command `git checkout HEAD 01-strings-tasks.js`

See https://stackoverflow.com/questions/7147270/hard-reset-of-a-single-file

I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
Selection_049

**Comment by lebedos2103** [comment]
I performed command "git checkout HEAD 01-strings-tasks.js", but nothing changed, i don't know, how to solve this problem(

**Comment by vramaniuk** [comment]
Try to revert this commit 
```git revert 352c5e36ab2b78ea733640169060ee24725880e3```

**Comment by vramaniuk** [comment]
And then update package.json again (update mocha)

File: task/07-yield-tasks.js [hunk 1]
```diff
@@ -33,7 +33,21 @@
  *
  */
 function* get99BottlesOfBeer() {
-    throw new Error('Not implemented');
+    let i = 99;
+    let flag = true;
+    while (i > 0 || !flag) {
+        let message = i === 0 ? "no more bottles" : i === 1 ? "1 bottle" : `${i} bottles`;
```
Comment (vramaniuk) [inline task/07-yield-tasks.js]: Please, avoid this code style, if-else is more readable than multiple conditional operators ? :

**Review by vramaniuk — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**